### ISSUE TITLE
Submodule error when cloning

### ISSUE THREAD

**Issue by 74monkeys** [chrisjwilliams/ska_tdt_ced#1]
fatal: no submodule mapping found in .gitmodules for path 'puppet/modules/zypprepo'

Given when trying to init submodules

**Comment by chrisjwilliams** [comment]
is this still here?

**Comment by 74monkeys** [comment]
Yes I believe it is. Just switched branch and I get the same error:

`fatal: no submodule mapping found in .gitmodules for path 'puppet/modules/zypprepo'
`
